README.md

```diff
@@ -5,12 +5,19 @@
 # Tesla Fleet Telemetry
 ---------------------------------
 
-Fleet Telemetry is a simple, scalable, and secure data exchange for device fleets.
+At Tesla we believe that security and privacy should be in the center of any modern technology. We want each and everyone of our customers to decide for themselves what data they share with third parties, how they share it and when it can be shared. We've developped a decentralized framework that allows customers to create a secure and direct brige from their Tesla devices to any provider they authorize.
```

Comment:
Slightly reduced verbosity:
```
At Tesla we believe that security and privacy is a core tenet of any modern technology. Customers should be able to decide what data they share with third parties, how they share it, and when it can be shared. We've developed a decentralized framework that allows customers to create a secure and direct bridge from their Tesla devices to any provider they authorize.
```